### PR TITLE
Implement AsLL ping responses

### LINKED ISSUE #9: modify ping/pong routine to its uplink include info for AsLL
newserv's ping/pong routine with its uplink current does not support the ping time or the info for AsLL. I think it might be nice if it did. It does not need to reply to remote AsLL requests, simply providing the needed info in the ping/pong routine should make the AsLL info available on its uplink. See example below.

```
xx.cc.quakenet.org AsLL for hub.xx.quakenet.org – RTT: 2ms Upstream: -28ms Downstream: 30ms
xx.cc.quakenet.org AsLL for services.cc.quakenet.org – [unknown]
xx.cc.quakenet.org AsLL for *: 2 local servers matched
```

(originally from wiebe)

### PR DESCRIPTION
**Pull request by CuleX**
This fixes #9.

You may want to double check this. I'm not 100% sure if I have the right numbers, but the values looked generally okay.

### CODE REVIEW

File: irc/irc.c [hunk 1]
```diff
@@ -611,9 +612,20 @@ void setnettime(time_t newtime) {
 }
 
 int handleping(void *sender, int cargc, char **cargv) {
-  if (cargc>0) {
-    irc_send("%s Z %s",mynumeric->content,cargv[cargc-1]);
-  }
+  char *ss = cargv[2], *sms = strchr(cargv[2], '.') + 1;
```
Comment (retropc) [inline irc/irc.c]: could you check cargc before looking at cargv[2]?

Comment (retropc) [reply irc/irc.c]: also if you could check the return value of strchr that would be grand

Comment (retropc) [reply irc/irc.c]: which bit aren't you sure about? :)

Comment (CuleX) [reply irc/irc.c]: The core arithmetic itself, in particular the mix of milliseconds and `tv_usec`. I think I've implemented it as ircu does, but it's always best to double check these details.

**Pull request closed without merging**